doc/Command-Reference.md

```diff
@@ -3358,6 +3360,41 @@ This command is the standard CMIS diagnostic control used for troubleshooting li
   admin@sonic:~$ sfputil debug loopback Ethernet88 media-side-output disable
   ```
 
+### CMIS debug rx-output
+
+This command enables or disables RX output on a specified port. It is a standard CMIS diagnostic control used for controlling rx-side link between the host switch and the transceiver module.
+
+**sfputil debug rx-ouput**
+
+- Usage:
+  ```
+  sfputil debug rx-output PORT_NAME <enable/disable>
+
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sfputil debug rx-output Ethernet88 enable
+  admin@sonic:~$ sfputil debug rx-output Ethernet88 disable
+  ```
+
+### CMIS debug tx-output
+
+This command enables or disables TX output on a specified port. It is a standard CMIS diagnostic control used for controlling tx-side link between the host switch and the transceiver module.
```

Comment:
tx output simply turns OFF the laser on the module

Reply:
fixed